pom.xml

```diff
@@ -112,7 +112,7 @@
     <sparkbundle.version>${spark2bundle.version}</sparkbundle.version>
     <flink.version>1.13.1</flink.version>
     <spark2.version>2.4.4</spark2.version>
-    <spark3.version>3.0.0</spark3.version>
+    <spark3.version>3.1.0</spark3.version>
```

Comment:
Spark 3.1.0 is broken, the minimal 3.1.x version is 3.1.1 https://spark.apache.org/news/next-official-release-spark-3.1.1.html

Reply:
BTW, any reason not to use the latest patch version?

Reply:
there is a 3.1.2 now as well?

Reply:
I would suggest always use the latest patched version, for this case, Spark 2.4.8, 3.0.3, 3.1.2

Reply:
Make sense to me.